crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1981,17 +1976,29 @@ impl TryFrom<&types::SetupMandateRouterData> for SetupIntentRequest {
             pm_type,
         ))?;
 
+        let mut meta_data = HashMap::from([
+            (
+                "metadata[txn_id]".to_string(),
+                format!("{}_{}_{}", item.merchant_id, item.payment_id, "1"),
```

Comment:
Please check the significance of this field. if there i none, Could you please remove this as we are hardcoding this.

Reply:
`metadata[txn_id]` is not significant. I am removing, the hardcoding.